package com.ExceptionHandling_task_4__21_8_2024;

public class Example11 {
    public static void main(String[] args) {
        try {
        } finally {
        }
		catch(Execption e)
        {
        }
    }
}
